docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+---
+version: '3'
+services:
+  plugin_test:
+    build:
+      context: .
+      dockerfile: ./tests/Dockerfile
+  plugin_lint:
+    image: buildkite/plugin-linter:latest
+    command: ['--id', 'monebag/monorepo-diff']
+    volumes:
+      - ".:/plugin"
```

Comment:
This was removed because it's unused and it's referencing monebag

Reply:
Yes @pzeballos